dev/AnimatedVisualPlayer/AnimatedVisualPlayer.cpp

```diff
@@ -560,8 +560,8 @@ winrt::Size AnimatedVisualPlayer::ArrangeOverride(winrt::Size const& finalSize)
 winrt::DependencyProperty InitializeDp(
     _In_ wstring_view const& propertyNameString,
     _In_ wstring_view const& propertyTypeNameString,
-    _In_opt_ winrt::IInspectable defaultValue,
-    _In_opt_ winrt::PropertyChangedCallback propertyChangedCallback = nullptr)
+    _In_opt_ const winrt::IInspectable& defaultValue,
+    _In_opt_ const winrt::PropertyChangedCallback& propertyChangedCallback = nullptr)
```

Comment:
awesome! I'm so excited to not remind people about this anymore.

Also can you have it scrub out SAL annotations? _In_ and _In_opt_ should be gone.

Reply:
I dont think there are any built in checks for that. we could write one possibly.

Reply:
I just did a find replace on all sal annotations.